types/dynamic_fee_tx.go

```diff
@@ -33,6 +33,22 @@ func (tx *DynamicFeeTx) gasPrice() *big.Int      { return nil }
 func (tx *DynamicFeeTx) gasTipCap() *big.Int     { return tx.GasTipCap }
 func (tx *DynamicFeeTx) gasFeeCap() *big.Int     { return tx.GasFeeCap }
 
+func (tx *DynamicFeeTx) effectiveGasPrice(baseFee *big.Int) *big.Int {
+	var tmp = new(big.Int)
```

Comment:
Nitpick, consistency wise

```suggestion
	tmp := new(big.Int)
```